### PR TITLE
Fix: Inner Section can’t be dragged into a column (#17249) [ED-5910]

### PR DESCRIPTION
**Pull request by YitzhakAvm**

### CODE REVIEW

File: assets/dev/js/editor/views/preview.js [hunk 1]
```diff
@@ -71,16 +71,12 @@ const Preview = BaseSectionsContainerView.extend( {
 	},
 
 	createElementFromModel: function( model, options = {} ) {
-		const shouldWrap = ! [ 'container', 'section' ].includes( model.elType || model?.get( 'elType' ) );
-
-		if ( ! shouldWrap ) {
-			delete options.at;
-		}
+		delete options.at;
```
Comment (iNewLegend) [inline assets/dev/js/editor/views/preview.js:74]: Why?

Comment (ItzhakAvm) [reply assets/dev/js/editor/views/preview.js:74]: If a json template is dragged (imported) into the document, and it contains sections with their `at` index, we want to ignore it and instead put the template in the position in which it was dragged into.

Comment (matipojo) [reply assets/dev/js/editor/views/preview.js:74]: In which case the `at` is saved to the JSON?

Comment (ItzhakAvm) [reply assets/dev/js/editor/views/preview.js]: @matipojo Sorry, I was wrong, it's not saved to JSON.
I removed this line.

**Review by matipojo — CHANGES_REQUESTED** [review]
Please cover with tests

**Comment by ItzhakAvm** [comment]
> Please cover with tests
@matipojo 
Done.

File: tests/qunit/tests/assets/dev/js/editor/document/elements/commands/create.spec.js [hunk 2]
```diff
@@ -298,6 +298,43 @@ export const Create = () => {
...
 			} );
 		} );
+
+		QUnit.module( 'Drag and Drop', () => {
+			QUnit.test( 'Widget: Inner Section into Column', ( assert ) => {
+				const eSection = ElementsHelper.createSection( 1 ),
+					eColumn = eSection.children[ 0 ],
+					eInnerSection = ( () => {
+						try {
+							return eColumn.view.createElementFromModel( {
```
Comment (matipojo) [inline tests/qunit/tests/assets/dev/js/editor/document/elements/commands/create.spec.js:308]: Use a command

**Review by matipojo — CHANGES_REQUESTED** [review]

**Pull request merged**